### ISSUE TITLE
Track time for terminal usage

### ISSUE THREAD

**Issue by rask** [wakatime/eclipse-wakatime#2]
I use this plugin with Aptana Studio (a web focused Eclipse version), and I do loads of work inside the GitBash embedded terminal window (git, node, compass, custom deployments, general terminal work, etc.).

Could there be any possibility to track time used inside an embedded terminal view inside Eclipse/Aptana?

I think the project inside which time is tracked could be determined with the current terminal working directory.

**Comment by alanhamlett** [comment]
Yes, we just need to find the corresponding event listener for Terminal events:

https://github.com/wakatime/eclipse-wakatime/blob/bf2bc60c78f41c6feeacdce0697ccf7fc379ad20/com.wakatime.eclipse.plugin/src/com/wakatime/eclipse/plugin/CustomEditorListener.java#L25

**Comment by gandarez** [comment]
@alanhamlett does this issue still make sense?
